src/system/documents/item.ts

```diff
@@ -375,12 +375,40 @@ export class CosmereItem<
         // Perform the roll
         const roll = await damageRoll(
             foundry.utils.mergeObject(options, {
-                formula: this.system.damage.formula,
+                formula: `${this.system.damage.formula}+${rollData.mod}`,
                 damageType: this.system.damage.type,
                 mod: rollData.mod,
                 data: rollData,
+                source: this.name,
             }),
         );
+        // We want to store the unmodded damage for use in graze calcs
+        // This isn't a particularly perfect solution, but it's functional
+        // only undoing the automatic addition of the selected attribute
+        rollData.baseRoll = (roll.total ?? 0) - (rollData.mod ?? 0);
+
+        // Roll the dice pool for graze damage silently if set.
+        let grazeRoll = undefined;
+        const grazeFormula =
+            this.system.damage.grazeOverrideFormula ?? `${rollData.baseRoll}`;
+        if (grazeFormula) {
+            grazeRoll = await damageRoll(
+                foundry.utils.mergeObject(options, {
+                    formula: grazeFormula,
+                    damageType: this.system.damage.type,
+                    data: rollData,
+                }),
+            );
```

Comment:
Since we're passing the base roll results as the formula for the graze, will that end up showing up in the chat message? I wonder if it's perhaps better to pass the actual evaluated terms? Or the damage formula (without mod) but then apply the dice results to the dice terms?

Reply:
Good thinking.
I guess the simple way would be to set the graze roll's dice property to that of the standard damage roll? That seems like it would be a solution about as simple as current? But I don't know if that actually works?
However I'm wondering if this is potentially just going to introduce unnecessary levels of branching, i.e. we'd need to first check if there isn't an override formula or the formula includes "the base roll string" then the formula used would have to get set based on which of those it was and the previous roll's dice inserted. Otherwise we just use the override formula as is?
I think the current approach covers the existing requirements and in conjunction with @MangoFVTT 's design discussions on discord, regardless of which damage pot you have selected for applying, when you expand the accordion to see the dice details it'll be showing both the default roll and the graze formula anyway.

Reply:
Why not pass the full roll object as `baseRoll`, then take the terms of baseRoll, remove the mods, and do Roll.fromTerms() on the remainder to get another roll object that we can pass as `grazeRoll`?  For non-override I mean.

Reply:
Yeah, that sounds positive, I don't really know the Roll API well enough to remember what's possible.
So if we have someone who wants to write an override formula for like the cremlings, say, which is `floor(@baseRoll/2)` will the foundry engine pick up the total from the roll object correctly, or will it need to evaluate to a numeric term?

Reply:
If we don't have a dice breakdown we can just not show the tooltip, which is fine. Obviously most people would prefer to HAVE the tooltip always, but if you use the Roll API that shouldn't be an issue. It does all the logic of evaluating terms internally afaik, including Function terms like the one you gave